services/BackgroundRunnerService.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import { log } from "@ledgerhq/logs";
+import {
+  withDevice,
+  withDevicePolling,
+} from "@ledgerhq/live-common/lib/hw/deviceAccess";
+import getDeviceInfo from "@ledgerhq/live-common/lib/hw/getDeviceInfo";
+import { from } from "rxjs";
+import { timeout } from "rxjs/operators";
+import { NativeModules } from "react-native";
+import { hasFinalFirmware } from "@ledgerhq/live-common/lib/hw/hasFinalFirmware";
+import prepareFirmwareUpdate from "@ledgerhq/live-common/lib/hw/firmwareUpdate-prepare";
+import mainFirmwareUpdate from "@ledgerhq/live-common/lib/hw/firmwareUpdate-main";
+import manager from "@ledgerhq/live-common/lib/manager";
+
+import { addBackgroundEvent } from "../src/actions/appstate";
+import { store } from "../src/context/LedgerStore";
+
+/**
+ * This task is not able to touch UI, but it will allow us to complete tasks
+ * even when the device goes to the background. We don't have access to hooks
+ * because we are not inside a component but we can read/write the store so we'll
+ * use that as the common-ground.
+ *
+ * Disclaimer: I might have missed some of the cases when
```

Comment:
You probably didn't miss any of the cases, delete this

Reply:
Deleted the whole file actually, as I did it in TS